components/collector/src/source_collectors/trivy/security_warnings.py

```diff
@@ -46,9 +56,11 @@ class TrivyJSONSecurityWarnings(SecurityWarningsSourceCollector, JSONFileSourceC
     def _parse_json(self, json: JSON, filename: str) -> Entities:
         """Override to parse the vulnerabilities from the Trivy JSON."""
         entities = Entities()
-        for dependency_repository in cast(TrivyJSON, json):
-            target = dependency_repository["Target"]
-            for vulnerability in dependency_repository.get("Vulnerabilities") or []:
+        trivy_json = cast(TrivyJSON, json)
+        results = trivy_json["Results"] if isinstance(trivy_json, dict) else trivy_json
```

Comment:
Pretty dirty that the `"Results"` section is only mentioned in the 3-year-old github discussion, but not on their official docs.

Reply:
Yeah, but now at least there's a ticket to improve the docs :-). 

https://github.com/aquasecurity/trivy/issues/7553